.travis.yml

```diff
@@ -9,12 +9,17 @@ python:
   - "2.7"
   - "3.5"
 
+env:
+  - TEST_DIR=root TEST_TYPE=lint
+  - TEST_DIR=root TEST_TYPE=unit
+  - TEST_DIR=utils
+
 install:
-  - pip install -r requirements.txt
```

Comment:
Indeed, I'm removing the The Ansible install directly from the tox deps for now, but when I add it back for the molecule testing, I will just do a `pip install` from the `commands` section.